pkg/cli/init-templates/.github/workflows/push.yaml

```diff
@@ -0,0 +1,50 @@
+name: Push to Replicate
+
+on:
+  # Workflow dispatch allows you to manually trigger the workflow from GitHub.com
+  # Go to your repo, click "Actions", click "Push to Replicate", click "Run workflow"
+  workflow_dispatch:
+    inputs:
+      model_name:
```

Comment:
Some models put this in `cog.yaml`. I'm not sure what kind of error `cog push` would generate if no model name is specified anywhere, though.

Reply:
Luckily Cog catches this case right away before proceeding:

```
$ cog push
ⅹ To push images, you must either set the 'image' option in cog.yaml or pass an image name as an argument. For example, 'cog push r8.im/your-username/hotdog-detector'
```